packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -341,6 +340,95 @@ describe('CheckoutModule', () => {
                 // Assert
                 expect(dispatch).toHaveBeenCalledWith(`${VUEX_CHECKOUT_ANALYTICS_MODULE}/updateAutofill`, state, { root: true });
             });
+
+            describe('when the `customer` model is not returned from the api', () => {
+                it('should not error when checking phone number', async () => {
+                    // Arrange
+                    checkoutDelivery.customer = null;
```

Comment:
We're changing a shared fixture here, it might cause errors when running in parrallel.

Reply:
You could mock the call from axios and return a falsey value or empty object too. Might be easier to read that way.